### PR TITLE
Create documentation

### PR DESCRIPTION
**Pull request by apoorvaeternity**
Go to the docs directory run `make html` to build the docs.

### CODE REVIEW

**Comment by apoorvaeternity** [comment]
To build locally go to the docs directory and write the command `make html`

File: docs/dashboard.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+=============
+The Dashboard
+=============
+
+The dashboard has been built using the Django web framework for python.
+It simply reads the dataset_details.json file generated by dashboard_script.py and shows it on the frontend.
```
Comment (henrykironde) [inline docs/dashboard.rst]: Let us delete this file and we add the contents into the docs/introduction.rst file.
Lets change the sentence

`
The dashboard has been built using the Django web framework for python.
It simply reads the dataset_details.json file generated by dashboard_script.py and shows it on the frontend.
`

**To**

The dashboard is built using the Django web framework for Python.
The Dashboard shows the status of each dataset's installation. The status for each dataset is stored in the `dataset_details.json` file  which is generated by the `dashboard_script.py` module

File: docs/developer.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
+=================
+Developer's guide
+=================
+
+About
+=====
+The Data Retriever Dashboard is divided into two parts:
+
+1. The Script
```
Comment (henrykironde) [inline docs/developer.rst]: lets us change  `The Script` >> The Helper script

File: docs/developer.rst [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+2. The Dashboard
+
+**The Script**
+The script(dashboard_script.py in retrieverdash/retrieverdash/dashboard_script/ directory) is run on every Sunday at 12:00 AM using cron job. It generates a
```
Comment (henrykironde) [inline docs/developer.rst]: is run on every Sunday at 12:00 AM using cron job.  >> is run regularly based on the cron job specification.

File: docs/developer.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+dataset_details.json file containing the details of the datasets and the diff files(html files showing
+side by side difference in the tables of a dataset if a change has been made to dataset).
```
Comment (henrykironde) [inline docs/developer.rst]: html files showing
side by side difference in the tables of a dataset if a change has been made to dataset)
>>
html files showing side by side diffs in the tables of a dataset if there is any change.

File: docs/developer.rst [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+locally on your machine. You’ll need Python 3.4+.
+
+**Steps to run Data Retriever Dashboard locally:**
+
+1. Clone the repository.
+2. From the directory containing manage.py, run the following command:
+   ``pip install -r requirements.txt`` to install the requirements for the dashboard.
+3. ``python manage.py makemigrations`` to make migrations.
+4. ``python manage.py migrate`` to migrate.
+5. ``python manage.py crontab add`` to add the cron job for running the script
```
Comment (henrykironde) [inline docs/developer.rst]: Could we be more specific on what we have to run when and why. We have the Helper script and the dashboard. 
So what should we run first.

I feel like steps 1-2 are fine but before we talk about step 3, we should a heading like
**To set up the dashboard and attach a cron job.**
3.
4.
5.

**To setup the dashboard and start running the job regardless of the set cron configuration.**
6....


change 7 to something like.
7. Open a browser and load the url that was provided when you run `python manage.py runserver`.

 (Note: When we are talking about `python manage.py runserver` in the point before 7, we should also mention that the url is default and if we want to change the port we could bind a new port number)

**Pull request merged**